stacker/lookups/handlers/output.py

```diff
@@ -17,11 +18,22 @@ def handler(value, provider=None, context=None, fqn=False, **kwargs):
         fqn (bool): boolean for whether or not the
             :class:`stacker.context.Context` should resolve the `fqn` of the
             stack.
+        rfqn (bool): boolean for whether or not the
+            :class:`stacker.context.Context` should resolve the `fqn` of the
+            stack prefixed by the namespace variable
 
     Returns:
         str: output from the specified stack
 
     """
+
+    if rfqn and context.environment.get('namespace'):
```

Comment:
Shouldn't need the `context.environment.get('namespace')` due to this check: https://github.com/remind101/stacker/blob/master/stacker/context.py#L49